prisondersdilemma.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May  7 13:37:28 2020
+
+@author: katha
+"""
+
+from psychopy.visual import Window, TextStim, Circle, ImageStim, Rect, ShapeStim
+from psychopy.core import wait, Clock
+from psychopy.event import getKeys, clearEvents, waitKeys, Mouse
+from numpy.random import shuffle
+import numpy as np
+import random
+
+#these are all just building blocks for now, not functioning code! it won't run yet
+
+#create window
+
+DISPSIZE = (1000, 700)
+BGC = 'white'
+FGC = (-1,-1,-1) #defining colour of text shown => black
+
+win = Window(size=DISPSIZE, units='pix', color = BGC, fullscr=False)
+
+
+#create objects and stimuli 
+
+background_image = ImageStim(win, 'prisonwall.png', autoDraw=True) #maybe for works for using a background image?
+welcome_str = f'Imagine you are one of two people being questioned about the same crime.\n'\
+'They are each talking to the interrogator separately. The interrogator gives each'\
+'person the same deal: \n'\
+'they can choose to vouch for the other person’s innocence (COOPERATING) or rat them out (DEFECTING).\n '\
+'And of course, there’s a twist. If both people vouch for each other, they’ll each get 3 months '\
+'off their sentence, but if the first person vouches for the second person, and the second '\
+'person rats them out, the first person will get no time off their sentence\n'\
+'and the second person will get 5 months off their time. Lastly, if they both '\
+'rat each other out, they each get 1 month off their time.\n'\
+'You can exit early by pressing Q\nPress space to start'
+
+sentCounter = TextStim(win, text='0 months', font="Arial", height=20, color='black', pos = (300, -300)) #counter stating amount of sentence
```

Comment:
To make your code more readable you can format it a bit better. For example, you can write it like this:

``` python
sentCounter = TextStim ( win,
                         text = '0 months',
                         font = 'Arial',
                         height = 20, 
                         color = 'black',
                         pos = (300, - 300)
                       )
```

Of course, this is just a 'cosmetic' suggestion 🙂

Reply:
That's a good suggestion and you already started implementing this below :-)